Source/vtkPyFR/vtkPyFRPipeline.cxx

```diff
@@ -645,31 +735,35 @@ int vtkPyFRPipeline::CoProcess(vtkCPDataDescription* dataDescription)
     this->InsituLink->InsituUpdate(dataDescription->GetTime(),
                                    dataDescription->GetTimeStep());
 
-    vtkUpdateFilter(this->Contour, dataDescription->GetTime());
-    vtkUpdateFilter(this->Slice, dataDescription->GetTime());
+      vtkUpdateFilter(this->Contour, dataDescription->GetTime());
+      vtkUpdateFilter(this->Slice, dataDescription->GetTime());
+
     if(this->PyData(dataDescription)->PrintMetadata()) {
-        double* bds = this->ContourMapper->GetBounds();
+        double* bds = this->ActiveMapper1->GetBounds();
```

Comment:
The mappers used to be ContourMapper and SliceMapper, so that we could in the future easily tweak the parameters on the mapper. So going back to that pattern seems reasonable.